modules/pushapk_scriptworker/files/requirements.txt

```diff
@@ -41,7 +41,7 @@ kiwisolver==1.0.1
 lxml==4.3.3
 matplotlib==3.0.3
 mohawk==0.3.4  # pyup: ignore
-mozapkpublisher==1.0.1
+mozapkpublisher==2.0.1
```

Comment:
Is mozapkpublisher really 2.0.1, or 2.0.0? Should be 2.0.0 unless we made a breaking change, made a release, then made a backwards-compatible fix release.

Reply:
I confirm it's 2.0.1. I busted 2.0.0 because of https://github.com/mozilla-releng/mozapkpublisher/pull/185